### DISCUSSION TITLE
[@types/busboy] Update busboy to v1.0.0+

### DISCUSSION THREAD

**Discussion by weilinzung** [DefinitelyTyped/DefinitelyTyped#57917] [Issues with a @types package]
https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/busboy

The latest version is v1.3 now.
https://www.npmjs.com/package/busboy

thanks!

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
@weilinzung PR is ready #57922
